tests/validator20/validate_apis_test.py

```diff
@@ -152,3 +152,82 @@ def test_api_check_default_fails(partial_parameter_spec, validator, instance):
     validation_error = excinfo.value.args[1]
     assert validation_error.instance == instance
     assert validation_error.validator == validator
+
```

Comment:
unrelated to line - looks like coverage failed?

Reply:
not sure what i was looking at when i posted this bit, gonna figure out how to drop

Reply:
O i was looking at the [travis CI page](https://travis-ci.org/Yelp/swagger_spec_validator/builds/386916806?utm_source=github_status&utm_medium=notification) looks like the latest commit is passing

Reply:
Yikes i don't know how to github, i shouldn't be the one approving the change, pls no merge